CHANGES.rst

```diff
@@ -212,7 +212,7 @@ astropy.utils
 
 - ``InheritDocstrings`` now also works on class properties. [#7166]
 
-- ``diff_values()`, ``report_diff_values()``, and ``where_not_allclose()``
+- ``diff_values()``, ``report_diff_values()``, and ``where_not_allclose()``
```

Comment:
Good catch. That was my bad. I wonder why CI didn't complain on my PR for this. Ops!

Reply:
Ah. I must have confused this with the what's new docs, which are parsed. Thanks for the clarification!